maps/aurora/aurora-4_mainlevel.dmm

```diff
@@ -63932,9 +63932,17 @@
 "mKz" = (
 /turf/simulated/wall,
 /area/maintenance/library)
+<<<<<<< HEAD
 "mKJ" = (
 /turf/simulated/floor/tiled,
 /area/crew_quarters/captain)
+=======
+"mKD" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/structure/trash_pile,
+/turf/simulated/floor/plating,
+/area/maintenance/research_port)
+>>>>>>> master
```

Comment:
You've accidentally committed some merge conflict resolution stuff. Need to deal with that. Not sure why that can even pass travis' tests.